.github/workflows/test.yml

```
@@ -0,0 +1,19 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+      - name: Install Node dependencies
+        run: yarn --frozen-lockfile
+      - run: yarn test
```

```suggestion
      - run: yarn test
      - run: yarn lint
```

From https://github.com/apple/ml-hierarchical-confusion-matrix/blob/main/package.json#L56=